backward.hpp

```diff
@@ -718,10 +744,15 @@ class StackTraceImplBase {
   }
 
   size_t skip_n_firsts() const { return _skip; }
+  void*  context() { return _context; }
+  void*  crash_addr() { return _crash_addr; }
+
 
 private:
   size_t _thread_id;
   size_t _skip;
+  void*  _context;
+  void*  _crash_addr;
```

Comment:
`crash_addr` -> `error_addr`

Reply:
Explained in the previous issue. I wanted to denote a difference between the function and the memory address. I can go back to _error_addr if you prefer.